src/node.rs

```diff
@@ -220,9 +220,9 @@ impl<S: StorageExt> SawtoothRaftNode<S> {
         }
     }
 
-    pub fn process_ready(&mut self) {
+    pub fn process_ready(&mut self) -> bool {
```

Comment:
It would be better to use an `enum` here to clarify what is being returned and support additional actions in the future. Something like:
```
enum ReadyStatus {
  Continue,
  Shutdown,
}
```

For example, if we add support later for changing raft settings while the network is running, we could add a `Reload` variant here.

Reply:
Here is a better write up on why this is a good idea: https://rust-lang-nursery.github.io/api-guidelines/type-safety.html?highlight=enum#arguments-convey-meaning-through-types-not-bool-or-option-c-custom-type

I highly recommend reading through that doc as it is full of good design ideas.